fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/functions/scalar/DateTrunc.java

```diff
@@ -91,4 +92,19 @@ public List<FunctionSignature> getSignatures() {
     public <R, C> R accept(ExpressionVisitor<R, C> visitor, C context) {
         return visitor.visitDateTrunc(this, context);
     }
+
+    @Override
+    public Expression withConstantArgs(Literal literal) {
+        return new DateTrunc(literal, child(1));
+    }
+
+    @Override
+    public int getMonotonicFunctionChildIndex() {
+        return 0;
+    }
+
+    @Override
+    public Monotonicity getMonotonicity() {
+        return new Monotonicity(true, true);
```

Comment:
maybe, u should use some static variables generated in Monotonicity class. refer to `org.apache.doris.nereids.properties.PhysicalProperties`